packages/theme/src/components/Tooltip/index.tsx

```diff
@@ -57,3 +56,42 @@ export const Tooltip = forwardRef<
     );
   }
 );
+
+export const AlternateTooltip = forwardRef<
```

Comment:
If you want to provide an alternate tooltip, I would specifically define a new component folder and name it relevant to when this component should be used. The components in `@repo/ui` are meant specifically as building blocks and not to satisfy *every* possible use case. I would move this to the frontend repo most likely!